### PR TITLE
Update index.php

### PR DESCRIPTION
**Pull request by CodingRule**
I removed the comments that were unnecessary or redundant. I also reorganized the code to follow a more logical flow. Finally, I simplified some of the variable assignments to make the code more concise.

### CODE REVIEW

File: public/index.php [hunk 1]
```diff
@@ -5,69 +5,25 @@
...
-*/
-
+// Handle the incoming request and send the response back to the browser
 $kernel = $app->make(Illuminate\Contracts\Http\Kernel::class);
-
-$response = $kernel->handle(
-    $request = Illuminate\Http\Request::capture()
-);
-
+$response = $kernel->handle(Illuminate\Http\Request::capture());
```
Comment (devnote-dev) [inline public/index.php:25]: Have you tested this? `$request` isn't a named parameter, it's a variable defined in the method call which is used below.

**Review by devnote-dev — COMMENTED** [review]
I don't think this PR is actually necessary, it can be quite useful to people familiarizing themselves with Pterodactyl's codebase. Regardless, one small point below.

File: public/index.php (hunk 1 above)
Comment (CodingRule) [reply public/index.php:25]: ohhh

**Comment by Jcodeerd** [comment]
I don't see the reason to remove most of the documentation/comments in the file.

**Comment by matthewpi** [comment]
This is copied directly from Laravel's template.  I don't see any reason to change any of this, we shouldn't be modifying this file anyways.

**Pull request closed without merging**